_docs/fixture-creation.md

```diff
@@ -97,20 +98,20 @@ fixture.Create<IReadOnlyCollection<string>>();
 fixture.Create<IReadOnlyDictionary<string, long>>();
 ```
 
-By default generated collections contain _many_ elements. The "many" size is controlled globally by the `fixture.RepeatCount` property. Currently the default is 3, but it's better to not rely on the exact value. If you need sequences with the exact number of elements, use the `CreateMany()` extension method.
+By default generated collections contain _many_ elements. The "many" size is controlled globally by the `fixture.RepeatCount` property. Currently the default is 3, but it's better to not rely on the exact value. If you need sequences with the exact number of elements, use the `CreateMany<T>()` extension method.
 
-## CreateMany<T>()
+### CreateMany&lt;T&gt;()
```

Comment:
Why do we avoid usage of `<T>` in header, while right below we use it in paragraph?

Reply:
In the header the markdown treats the angle brackets as a definition of a HTML tag and then it doesn't display it. The other uses of the `<T>` are placed inside code blocks which reserves the original syntax.
![image](https://user-images.githubusercontent.com/7030530/89398321-9f00b780-d719-11ea-9a33-1d69e7ac9104.png)

Reply:
Got it! Thanks for explanation 😊